kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
@singhpratyush , please go through this. This is the exact logs from creation of server to the error I got. Please see where I am going wrong. This time I didn't use deploy script, instead I wrote them as you suggested.
https://gist.github.com/vibhcool/58ed3f11ee504a6760c123d6beeba7f5

Server deployment: http://35.189.73.55/